dataland-frontend/tests/e2e/specs/sfdr/DataIntegrity.ts

```diff
@@ -61,7 +62,8 @@ describeIf(
         const uniqueCompanyMarker = Date.now().toString();
         const testCompanyName = "Company-Created-In-Sfdr-DataIntegrity-Test-" + uniqueCompanyMarker;
         getKeycloakToken(admin_name, admin_pw).then((token: string) => {
-          return uploadCompanyAndSfdrDataViaApi(
+          return uploadCompanyAndFrameworkData(
+            "sfdr",
```

Comment:
Du hast das für lksg zu DataTypeEnum.lksg geändert. Sollten wir da sdann überall machen?

Reply:
Habe ich gemäß dem TODO getan.   Es sollte nirgendwo mehr ein literal wie "sfdr" als Parameter in der Methode benutzt werden.

Außerdem habe ich alle "uploadOne***DatasetViaApi" mit der neuen Methode "uploadFrameworkData" ersetzt.

Reply:
Top. Dankeschön. Ich check das kurz und danach mache ich hier zu